### DISCUSSION TITLE
Write mesh to STL format file

### DISCUSSION THREAD

**Discussion by esuig** [CGAL/cgal#6860] [Q&A] [answered]
Hello, 

I have a polyedron (CGAL::Polyhedron_3<K>); I would like to write it to a file in .STL format. I know the function CGAL::IO::Write_STL() exists, but how can I feed the function starting from my polyedron?

**Comment by afabri** [comment, marked as answer]
What do you mean with "feeding the function"?    When you have a `CGAL::Surface_mesh<CGAL::Simple_cartesian<double>::Point_3 > sm;`  you just call `CGAL::IO::write_STL(std::cout, sm)`.   Similar for a `CGAL::Polyhedron_3<CGAL::Simple_cartesian<double>>`.

**Reply by esuig** [reply]
Hello, thanks for your reply and info, that solved my problem
